packages/gestalt-datepicker/dist/gestalt-datepicker.js.flow

```diff
@@ -0,0 +1,2 @@
+// @flow strict-local
```

Comment:
Does this need to be strict-local since there aren't any dependencies in here?

Reply:
Fixed.